scripts/init.sh

```diff
@@ -131,6 +135,7 @@ start-frequency-instant)
     --rpc-external \
     --rpc-cors all \
     --rpc-methods=Unsafe \
+    # --pool-type=fork-aware \  Fork aware pool is not supported with instant sealing
     $offchain_params \
     --tmp
   ;;
```

Comment:
At least on my mac, the comment caused the command to ignore all the other parameters
```suggestion
    $offchain_params \
    --tmp
  ;;
  # Fork aware pool is not supported with instant sealing
```